ci-scripts/publish.sh

```diff
@@ -19,6 +19,8 @@ if [[ "$TRAVIS_BRANCH" = "tmp_branch_for_automated_release_do_not_use" ]]; then
   
     npm publish
 
+    npm run deploy
```

Comment:
this is covered in the if statement above right? only push on actual releases, not RCs?

Reply:
Yeah, only publish on `npm run release`